### DISCUSSION TITLE
interview/mysql-60

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#189] [Announcements]
# interview/mysql-60

Java程序员进阶之路，小白的零基础Java教程，60 道 MySQL 精选面试题👍

https://tobebetterjavaer.com/interview/mysql-60.html

**Comment by studeyang** [comment]
你是看过 MySQL45讲 的。

**Comment by XuX-jeeee** [comment]
虽然算是学过了mysql，但看起来还是好多盲点，一些名词都不认识的，面试真的会考这么多吗。

**Comment by pengyuanqiuqiu** [comment]
17的redolog和binlog日志类型是错误，redolog是物理日志，binlog是逻辑日志
